dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
   end
 
   def k5_public_map_index
-    @workshops = @workshops.where(
+    @workshops = Pd::Workshop.where(
```

Comment:
Given that we are opening this up, does it make sense to whitelist the fields being returned to the map?